source/user-manual/wazuh-dashboard/navigating-the-wazuh-dashboard.rst

```diff
@@ -40,7 +40,6 @@ This section shows dashboards for:
 -  :doc:`Threat Hunting </getting-started/use-cases/threat-hunting>`
 -  :doc:`Vulnerability Detection </user-manual/capabilities/vulnerability-detection/index>`
 -  :doc:`MITRE ATT&CK </user-manual/ruleset/mitre>`
--  :doc:`VirusTotal </user-manual/capabilities/malware-detection/virus-total-integration>`
```

Comment:
Do not remove this; it is related to the issue: "Update Malware detection documentation to include VirusTotal."